Makefile.am

```diff
@@ -188,14 +188,11 @@ BATS = \
 	test/functional/update/update-verify-fix-path-hash-mismatch.bats \
 	test/functional/update/update-verify-fix-path-missing-dir.bats \
 	test/functional/update/update-verify-fullfile-hash.bats \
-	test/functional/verify/verify-add-missing-directory.bats \
```

Comment:
The only thing I couldn't track on this pull request is where are the tests that replaces the tests in verify-add-missing-directory and verify-check-missing-directory

Reply:
yes, I removed those tests because I added the tests in verify-missing-file: https://github.com/clearlinux/swupd-client/pull/673/files#diff-6d1105117307cb164dc89f95a6ee4110, and those tests include a missing file and a missing directory. In those tests file _/foo/bar_ is missing, so directory _/foo_ has to be added before file _bar_ can be added.

Reply:
nice.